recipes/jsonnet/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,19 @@
+#include <libjsonnet++.h>
+#include <libjsonnet.h>
+
+#include <iostream>
+
+int main() {
+  // C++ library
+  jsonnet::Jsonnet j;
+  if (!j.init()) {
+    return 1;
+  }
+  std::cout << jsonnet::Jsonnet::version() << "\n";
```

Comment:
```suggestion
  std::cout << jsonnet::Jsonnet::version() << "\n";
```

It's not available for GCC5 + libstdc++. Only works with libstdc++11

Besides that, both libjsonnet++.so and libjsonnet.so require libstdc++

ldd lib/libjsonnet.so               
	linux-vdso.so.1 =>  (0x00007ffde2908000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f8b93523000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f8b9321a000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f8b93004000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f8b92c3a000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f8b93b75000)
	
ldd lib/libjsonnet++.so
	linux-vdso.so.1 =>  (0x00007ffc61d7f000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fa4dc271000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fa4dc05b000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fa4dbc91000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fa4db988000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fa4dc7f7000)

Reply:
what exactly is not available? I don't see what have you changed in your suggestion